srcalc/src/main/webapp/WEB-INF/views/adminHome.jsp

```diff
@@ -33,5 +33,27 @@
         <a href="${newDiscreteVar}" class="btn-default">Discrete Numerical</a></li>
     </ul>
     </div>
+    <h3>Rules</h3>
+    <table class="srcalcTable" id="rulesTable">
+    <tr>
+        <th class="main">Name</th>
+        <th>Edit</th>
+    </tr>
+    <c:forEach var="rule" items="${rules}">
+        <c:url var="editRule" value="/admin/rules/${rule.id}" />
+	    <tr>
+	        <td><c:out value="${rule.displayName}"/></td>
+	        <td><a href="#" class="editObjectLink">Edit</a></td>
```

Comment:
You don't need to change this, but it would be better not to make this a link at all.